### ISSUE TITLE
[Bug] accessibilityRole in Header not variable

### ISSUE THREAD

**Issue by Abhisflyingsoon** [wix/react-native-calendars#2526]
## Description

Accessibility property "accessibilityRole" in Header should not be hardcoded to "adjustable". It should be variable.
Noticed weird behaviour, when scrolling with one finger while the header was focused. It somehow ignored every boundary I set and showed duplicate months in a list. Changing the accessibilityRole fixed the issue for us.
